src/js/controllers/chatController.js

```diff
@@ -62,51 +62,46 @@ module.exports = ViewController.extend({
     },
 
     sendMessage: function(text) {
-        return $.Deferred().resolve().then(function() {
-            var promise = $.Deferred();
-
-            if (this.conversation.isNew()) {
-                this.conversation = this.collection.create(this.conversation, {
-                    success: promise.resolve,
-                    error: promise.reject
-                });
-            } else {
-                promise.resolve(this.conversation);
-            }
-
-            return promise;
-        }.bind(this))
```

Comment:
Ah ok. Makes sense.

But yeah, if we could use ES6, this would be a good place to use arrow functions.